next.config.js

```diff
@@ -11,6 +11,41 @@ const mdx = withMDX({
   pageExtensions: ['ts', 'tsx', 'js', 'jsx', 'md', 'mdx'],
 })
 
+const cleanUrl = (url) => {
```

Comment:
Maybe this function could have a more specific name? Seeing what it does, how about `getHostname`? Or if we wanna be super precise `getProtocolAndHostname`?